CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 We want to make contributing to this project as easy and transparent as
 possible.
 
+## Code of Conduct
```

Comment:
Just for my education, maybe because I don't know the process -- but couldn't this just link to `CODE_OF_CONDUCT.md` instead of repeating the same wording again?

Reply:
Sure - my thought was that it would then take them 2 clicks to get to the actual COC text, since the`CODE_OF_CONDUCT.md` asks them to click through to the cannonical COC. But it is weird to have exactly the same wording, so I can update them. Will do that later.

Reply:
**Or** if you have time, edit this PR as you wish and then I can update my other PRs to match. :)

Reply:
Done. Does that look reasonable?